File: FirstApp/Messenger/MessengerForIndividual.cs
using System;
using System.Collections.Generic;
using System.Drawing;

namespace FirstApp
{
    public partial class MessengerForIndividual
    {

        public List<Kitty> SelectedKitties { get; set; } = new List<Kitty>();

        public MessengerForIndividual()
        {
            InitializeComponent();
        }

        private void MessengerForIndividual_Load(object sender, EventArgs e)
        {
            ProjectModule.InitaliseConnection();
            try
            {
                ProjectModule.myconnection.Open();
            }
            catch
            {
            }

            if (SelectedKitties.Count == 1)
            {
                if (SelectedKitties[0].GetInstalmentsPending() > 0)
                {
                    TemplateBT.PerformClick();
                }
                else
                {
                    ReceiptBT.PerformClick();
                }
            }
            else
            {
                ReceiptBT.PerformClick();
            }

        }

        private void ReceiptBT_Click(object sender, EventArgs e)
        {
            TemplateBT.IconColor = Color.White;
            ReceiptBT.IconColor = Color.Lime;
            var x = new RecieptMessenger()
            {
                ListOfKitties = SelectedKitties,
                TopMost = false,
                TopLevel = false
            };
            Panel1.Controls.Clear();
            Panel1.Controls.Add(x);
            x.Show();
        }

        private void TemplateBT_Click(object sender, EventArgs e)
        {
            ReceiptBT.IconColor = Color.White;
            TemplateBT.IconColor = Color.Lime;
            var x = new MessageAndImageSender()
            {
                SelectedKitties = SelectedKitties,
                TopMost = false,
                TopLevel = false
            };
            Panel1.Controls.Clear();
            Panel1.Controls.Add(x);
            x.Show();
        }
    }
}

File: OrnaTrack/Messenger/MessengerForIndividual.vb
Public Class MessengerForIndividual

    Public Property SelectedKitties As New List(Of Kitty)

    Private Sub MessengerForIndividual_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        InitaliseConnection()
        Try : myconnection.Open() : Catch : End Try

        If SelectedKitties.Count = 1 Then
            If SelectedKitties(0).GetInstalmentsPending > 0 Then
                TemplateBT.PerformClick()
            Else
                ReceiptBT.PerformClick()
            End If
        Else
            ReceiptBT.PerformClick()
        End If

    End Sub

    Private Sub ReceiptBT_Click(sender As Object, e As EventArgs) Handles ReceiptBT.Click
        TemplateBT.IconColor = Color.White
        ReceiptBT.IconColor = Color.Lime
        Dim x As New RecieptMessenger With {
            .ListOfKitties = SelectedKitties,
            .TopMost = False,
            .TopLevel = False
        }
        Panel1.Controls.Clear()
        Panel1.Controls.Add(x)
        x.Show()
    End Sub

    Private Sub TemplateBT_Click(sender As Object, e As EventArgs) Handles TemplateBT.Click
        ReceiptBT.IconColor = Color.White
        TemplateBT.IconColor = Color.Lime
        Dim x As New MessageAndImageSender With {
            .SelectedKitties = SelectedKitties,
            .TopMost = False,
            .TopLevel = False
        }
        Panel1.Controls.Clear()
        Panel1.Controls.Add(x)
        x.Show()
    End Sub
End Class